# Stores a detected landmark pose with covariance as well as a known landmark pose
geometry_msgs/PoseWithCovarianceStamped detected_pose
geometry_msgs/PoseStamped expected_pose
